src/components/layout/Header.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Header = () => {
+    return (
+        <div className='header'>
```

Comment:
```suggestion
        <header className='header'>
```

This is the same kind of change that needs making to the other components as well. Find the divs marking the nav, main and footer and replace the divs with the semantic version of the tag.